src/CommunityToolkit.Maui.UnitTests/CommunityToolkit.Maui.UnitTests.csproj

```diff
@@ -5,7 +5,7 @@
     <UseMaui>true</UseMaui>
     <IsPackable>false</IsPackable>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
-    <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
+    <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GF</CompilerGeneratedFilesOutputPath>
```

Comment:
I know the reason behind this change, but can you do it in another PR? That way will be easy to us to track it in the future, if needed